server/api/utils/auth.js

```diff
@@ -116,9 +116,10 @@ module.exports = expressJwtPolicies({
    * @param {function} next - Function that calls the next middleware in the stack.
    * @see https://www.npmjs.com/package/express-jwt-policies
    */
-  authorizationErrorHandler: function(err, req, res, next) {
-    if (err.status != 403) {
-      return next(err);
+  authorizationErrorHandler(err, req, res, next) {
+    if (err.status !== 403) {
+      next(err);
+      return;
```

Comment:
Why the splitting here ?

Reply:
Otherwise the JSDoc rule complains that the return value is not documented. So I can either split this line, or add a `@returns {undefined}`. I don't really care either way. I think it's the only such case I found.